plugin/case-reporting/src/main/java/org/opencds/cqf/ruler/casereporting/r4/MeasureDataProcessProvider.java

```diff
@@ -149,9 +132,9 @@ private void gatherPatientsFromReport(MeasureReport report, Map<String, Referenc
 			logger.debug("No subject results found.");
 			return results;
 		}
-		IBaseResource baseList = myDaoRegistry.getResourceDao(subjectResults.getReferenceElement().getResourceType()).read(subjectResults.getReferenceElement());
+		IBaseResource baseList = read(subjectResults.getReferenceElement());
 		if (baseList == null) {
-			logger.debug(String.format("No subject results found for: %s", subjectResults.getReference()));
+			logger.debug("No subject results found for: {}", subjectResults.getReference());
```

Comment:
There was quite a bit of this too. The loggers accept a string format and parameters, so no need to pre-format it.